### PR TITLE
chore: Improve PR template

### PR DESCRIPTION
**Pull request by balanza**
### Motivations
For authors: help to provide an insightful glimpse on the proposed changes.
For readers: understand the PR without the need to dig into file changes
For reviewers: have consistency between proposals  

The template proposes two different sections based on the kind of change the PR introduces.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+- [CHORE] for PRs that propose changes to the infrastructure of the radar such as scripts, versioning, templates.
 
-When making a pull request, please ensure that the following
-things are done.
+🙏 Please use the relevant template section, it will help you to better communicate the proposed changes.
+⚠️ Be sure you remove the unused section
+-->
+<!-- SECTION: CONTENT -->
+<!--
+Required. Help reviewers with a hint on what this PR is about. Fill the table as per the following instructions:
```
Comment (gunzip) [inline .github/pull_request_template.md:13]: I'd remove this part. What we expect is to have PR that add content (markdown) which will be self-documenting.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+item title -> the title of the element subject of this PR
+change type -> either `add`, `remove`, `move` (change ring) or `edit` (any other modifications)
+-->
+|item title|change type|
+|-|-|
+|<!-- title here -->|<!-- change here -->|
```
Comment (gunzip) [inline .github/pull_request_template.md:19]: ditto

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+- [CONTENT] for PRs that propose changes to content
+- [CHORE] for PRs that propose changes to the infrastructure of the radar such as scripts, versioning, templates.
 
-When making a pull request, please ensure that the following
-things are done.
+🙏 Please use the relevant template section, it will help you to better communicate the proposed changes.
+⚠️ Be sure you remove the unused section
+-->
+<!-- SECTION: CONTENT -->
+<!--
```
Comment (gunzip) [inline .github/pull_request_template.md:12]: I'd remove this part. What we expect from CODEOWNERS is to contribute to content so I'd keep the template as lean as possible. About fixes to UI and scripts, we can manage them without too much burocracy.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
+-->
+|item title|change type|
+|-|-|
+|<!-- title here -->|<!-- change here -->|
+
+### Motivations
+<!-- 
+Optional. In case you want to add more context on why the proposed changes might be considered.
+Remove the section if not needed
+-->
```
Comment (gunzip) [inline .github/pull_request_template.md:25]: The context will be always the same (adding new entries) and self-evident.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -13,3 +39,13 @@ things are done.
...
+
+<!-- END SECTION: CONTENT -->
+
+<!-- SECTION: CHORE -->
+### Motivations
+<!-- 
+Required. Add more context on why the proposed changes might be considered.
+-->
+
+<!-- END SECTION: CHORE -->
```
Comment (gunzip) [inline .github/pull_request_template.md:51]: ditto

Comment (gunzip) [reply .github/pull_request_template.md:51]: I'd avoid to put further burden on contributors keeping the template as the bare minum and focused on content writing.

**Comment by balanza** [comment]
I see no interest in these changes

**Pull request closed without merging**